home/import_content_pages.py

```diff
@@ -216,11 +191,11 @@ def add_go_to_page_list_items(self) -> None:
                         row = self.shadow_pages[(slug, locale)]
                         raise ImportException(
                             f"No pages found with slug '{item['slug']}' and locale "
-                            f"'{locale}' for go_to_page list item '{item['title']}' on "
+                            f"'{locale}' for go_to_page {item_type[:-1]} '{item['title']}' on "
                             f"page '{slug}'",
                             row.row_num,
                         )
-                    page.whatsapp_body[message_index].value["list_items"].insert(
+                    page.whatsapp_body[message_index].value[item_type].insert(
```

Comment:
If this is the only place we care about whether we have a button or a list item, we can probably include this key in the data structure we put in `items_dict` and consolidate things even further. However, I've spent over an hour trying to figure out how all the button/list stuff fits together so I can start looking into this consolidation and not had much success, so I think let's leave it as-is for now and try fix it separately.